src/main/AndroidManifest.xml

```diff
@@ -376,6 +376,13 @@
               android:exported="true">
     </activity>
 
+    <activity android:name=".WebxdcStoreActivity"
+              android:label=""
```

Comment:
The label should probably be `R.string.webxdc_apps`

Reply:
I just removed it, it is not present for FullMsgActivity and other some activities, I copied from WebxdcActivity where it is set to empty for some reason but probably also not necessary